website/assets/js/pages/customers/new-license.page.js

```diff
@@ -0,0 +1,101 @@
+parasails.registerPage('new-license', {
+  //  ╦╔╗╔╦╔╦╗╦╔═╗╦    ╔═╗╔╦╗╔═╗╔╦╗╔═╗
+  //  ║║║║║ ║ ║╠═╣║    ╚═╗ ║ ╠═╣ ║ ║╣
+  //  ╩╝╚╝╩ ╩ ╩╩ ╩╩═╝  ╚═╝ ╩ ╩ ╩ ╩ ╚═╝
+  data: {
+    // Form data
+    formData: { /* … */ },
+
+    // For tracking client-side validation errors in our form.
+    // > Has property set to `true` for each invalid property in `formData`.
+    formErrors: { /* … */ },
+
+    quoteFormRules: {
+      numberOfHosts: {required: true},
+    },
+
+    billingFormRules: {
+      paymentSource: {required: true},
+    },
+
+    // Syncing / loading state
+    syncing: false,
+
+    // Server error state
+    cloudError: '',
+
+    // Success state when form has been submitted
+    cloudSuccess: false,
```

Comment:
also, it looks like you're using a state variable called `orderComplete` instead of `cloudSuccess`, so I think you should verify that `cloudSuccess` is indeed unused in the HTML, then remove it from the .page.js file